### ISSUE TITLE
load_contracts.py does not upload all contracts

### ISSUE THREAD

**Issue by tinybike** [AugurProject/augur-core#62]
```
compiling /home/jack/src/augur-core/src/functions/quicksort.se
compiling /home/jack/src/augur-core/src/functions/insertionsort.se
compiling /home/jack/src/augur-core/src/functions/comments.se
compiling /home/jack/src/augur-core/src/data_api/fxpFunctions.se
compiling /home/jack/src/augur-core/src/data_api/cash.se
compiling /home/jack/src/augur-core/src/data_api/branches.se
compiling /home/jack/src/augur-core/src/data_api/events.se
compiling /home/jack/src/augur-core/src/data_api/expiringEvents.se
compiling /home/jack/src/augur-core/src/data_api/info.se
compiling /home/jack/src/augur-core/src/data_api/markets.se
compiling /home/jack/src/augur-core/src/functions/closeMarketTwo.se
compiling /home/jack/src/augur-core/src/functions/closeMarketFour.se
compiling /home/jack/src/augur-core/src/functions/closeMarketEight.se
compiling /home/jack/src/augur-core/src/functions/transferShares.se
compiling /home/jack/src/augur-core/src/functions/closeMarketOne.se
```
...where are the rest?  (The amount of grief this script has caused is really astonishing when you stop and think about it.)

**Comment by tinybike** [comment]
Aha, there's a circular dependency between makeReports and reporting....

**Comment by tinybike** [comment]
Fixed in 68ed80af08e0930da83ff0a396a858f817fe3342.
